src/Notes/WC_Admin_Notes_Facebook_Extension.php

```diff
@@ -97,6 +97,25 @@ public function install_facebook_extension( $note ) {
 
 			$activate_request = array( 'plugins' => 'facebook-for-woocommerce' );
 			$installer->activate_plugins( $activate_request );
+
+			$content = __( 'You\'re almost ready to start driving sales with Facebook. Complete the setup steps to control how WooCommerce integrates with your Facebook store.', 'woocommerce-admin' );
+			$note->set_title( __( 'Market on Facebook — Installed', 'woocommerce-admin' ) );
```

Comment:
When this note replaces the installation note, I get the following console error:

<img width="1052" alt="Screen Shot 2019-10-23 at 12 54 08 PM" src="https://user-images.githubusercontent.com/10561050/67360147-b9f1a900-f597-11e9-8dd3-c933f4e80803.png">

Reply:
Fixed in acc7b6f.